### PR TITLE
test: fix leak test

### PR DESCRIPTION
**Pull request by lysu**
make sure close dom/store in all test.

## What have you changed? (mandatory)

fix test leak in test case.

## What are the type of the changes (mandatory)?

- test fix

## How has this PR been tested (mandatory)?

modification is test.

### CODE REVIEW

File: util/ranger/ranger_test.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func (s *testRangerSuite) SetUpSuite(c *C) {
 	s.Parser = parser.New()
 }
 
-func newStoreWithBootstrap(c *C) (kv.Storage, error) {
+func newStoreWithBootstrap(c *C) (*domain.Domain, kv.Storage, error) {
```
Comment (shenli) [inline util/ranger/ranger_test.go]: s/newStoreWithBootstrap/newDomainStoreWithBootstrap/

Comment (lysu) [reply util/ranger/ranger_test.go]: Done～

**Comment by shenli** [comment]
LGTM

File: util/ranger/ranger_test.go [hunk 2]
```diff
@@ -627,8 +636,11 @@ func (s *testRangerSuite) TestIndexRangeForUnsignedInt(c *C) {
 
 func (s *testRangerSuite) TestColumnRange(c *C) {
 	defer testleak.AfterTest(c)()
-	store, err := newStoreWithBootstrap(c)
-	defer store.Close()
+	dom, store, err := newDomainStoreWithBootstrap(c)
```
Comment (zz-jason) [inline util/ranger/ranger_test.go]: maybe we can implement function `TearDownTest` for `testRangerSuite` and put the code deferred into that function.

Comment (lysu) [reply util/ranger/ranger_test.go]: This will modify too much, this is a quick fix...so may we do it later~?

Comment (zz-jason) [reply util/ranger/ranger_test.go]: ok

**Comment by tiancaiamao** [comment]
Good job! @lysu

**Pull request merged**